src/lib/swapper/swappers/OneInchSwapper/utils/fetchOneInchSwap.ts

```diff
@@ -15,7 +14,10 @@ export type FetchOneInchSwapInput = {
   receiveAddress: string
   sellAmountBeforeFeesCryptoBaseUnit: string
   sellAsset: Asset
-  maximumSlippageDecimalPercentage?: string
+  // The maximum acceptable slippage percentage for the swap (e.g., 1 for 1%)
+  // Note, we internally represent slippage as a decimal across the app (e.g., 0.01 for 1%)
+  // so we need to multiply it by 100 when calling 1inch swap endpoint
+  maximumSlippageDecimalPercentage: string
```

Comment:
this comment is confusing because its on the input to the swapper rather than the input to the 1inch api call.

Reply:
https://github.com/shapeshift/web/pull/5023/commits/23d8772d13a7a5b2b1e0d0398ef04bcf2dc481e0